examples/case_params_with_units.py

```diff
@@ -69,3 +70,9 @@
 
 params_as_json = params_solver.to_flow360_json()
 print(params_as_json)
+
+# Class name removal from loc
+with fl.SI_unit_system:
```

Comment:
validation should not be called inside context

Reply:
This fails without the context - in case-editor branch we have the unit system context passed as an argument, but on develop this is the only way to prevent the "Params must be created within a unit system context (...)" error message. Once changes from case-editor are applied this can be discarded.